doc/api/cli.md

```diff
@@ -863,6 +863,20 @@ Use this flag to generate a blob that can be injected into the Node.js
 binary to produce a [single executable application][]. See the documentation
 about [this configuration][`--experimental-sea-config`] for details.
 
+
+### `--experimental-task-ordering`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Enable experimental task ordering. Always drain micro task queue
```

Comment:
```suggestion
Enable experimental task ordering. Always drain Microtasks queue
```

It would also be useful to link to a doc like https://developer.mozilla.org/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide#microtasks here.